### PR TITLE
1526 import org units

### PR DESCRIPTION
**Pull request by IgorNadj**
### Changes:

- Fix lat/lon of a city
- Copy districts and sub_districts from laos_schools

---

### CODE REVIEW

File: packages/database/src/migrations/20210201051217-SyncLaosEocEntitiesWithLaosSchools-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+    const subDistrictId = await getEntityId(db, province.capital.subDistrictCode);
+
+    const cityId = await getEntityId(db, province.capital.code);
+
+    await db.runSql(`UPDATE entity SET parent_id = '${subDistrictId}' WHERE id = '${cityId}';`);
+
+    await db.runSql(`
+      INSERT INTO entity_relation (id, parent_id, child_id, entity_hierarchy_id)
+      VALUES (generate_object_id(), '${subDistrictId}', '${cityId}', '${laosEocEntityHierarchyId}');
```
Comment (kael89) [inline packages/database/src/migrations/20210201051217-SyncLaosEocEntitiesWithLaosSchools-modifies-data.js:178]: I guess we also need to delete existing entity relations?

**Review by kael89 — COMMENTED** [review]
Great work, your migration was a pleasure to read 😃 Just one question before I approve

File: packages/database/src/migrations/20210201051217-SyncLaosEocEntitiesWithLaosSchools-modifies-data.js (hunk 1 above)
Comment (IgorNadj) [reply packages/database/src/migrations/20210201051217-SyncLaosEocEntitiesWithLaosSchools-modifies-data.js:182]: Ah very good spotting! I was messing with data manually and missed that step. Updated

**Review by kael89 — APPROVED** [review]
🙌

**Pull request merged**